Jellyfin.Plugin.Anime/Providers/KitsuIO/KitsuIoExternalId.cs

```diff
@@ -0,0 +1,26 @@
+using MediaBrowser.Controller.Entities.Movies;
+using MediaBrowser.Controller.Entities.TV;
+using MediaBrowser.Controller.Providers;
+using MediaBrowser.Model.Entities;
+using MediaBrowser.Model.Providers;
+
+namespace Jellyfin.Plugin.Anime.Providers.KitsuIO
+{
+    public class KitsuIoExternalId : IExternalId
+    {
+        public bool Supports(IHasProviderIds item)
+            => item is Series || item is Movie;
+        
+        public string ProviderName
+            => "KitsuIO";
```

Comment:
```suggestion
            => "Kitsu";
```
I've personally never heard Kitsu refered to as KitsuIO before.